roles/leongold.bind_to_vfio/tasks/main.yml

```diff
@@ -0,0 +1,3 @@
+- name: bind devices to vfio
```

Comment:
filenames mentions leongold. I don't suppose this is intentional.

Reply:
Done